systemtests/tests/py2plug-fd-local-fileset-basic/CMakeLists.txt

```diff
@@ -17,24 +17,26 @@
 #   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 #   02110-1301, USA.
 
-if(TARGET python-fd)
-  create_systemtest(${SYSTEMTEST_PREFIX} "py2plug-fd-local-fileset-basic")
+get_filename_component(BASENAME ${CMAKE_CURRENT_BINARY_DIR} NAME)
+
+if(TARGET python-fd AND BASENAME MATCHES "py2")
+  create_systemtest(${SYSTEMTEST_PREFIX} ${BASENAME})
 else()
-  create_systemtest(
-    ${SYSTEMTEST_PREFIX} "py2plug-fd-local-fileset-basic" DISABLED
-  )
+  create_systemtest(${SYSTEMTEST_PREFIX} ${BASENAME} DISABLED)
 endif()
 
-if(TARGET python3-fd)
-  create_systemtest(${SYSTEMTEST_PREFIX} "py3plug-fd-local-fileset-basic")
-  # Fails with UnicodeEncodeError: 'utf-8' codec can't encode characters in
+if(TARGET python3-fd AND BASENAME MATCHES "py3")
+  create_systemtest(${SYSTEMTEST_PREFIX} ${BASENAME})
+  # Fail with UnicodeEncodeError: 'utf-8' codec can't encode characters in
   # position 133-142: surrogates not allowed
   set_tests_properties(
-    "${SYSTEMTEST_PREFIX}py3plug-fd-local-fileset-basic" PROPERTIES LABELS
-                                                                    "broken"
+    "${SYSTEMTEST_PREFIX}py3plug-fd-local-fileset-basic:basic"
+    PROPERTIES LABELS "broken"
   )
-else()
-  create_systemtest(
-    ${SYSTEMTEST_PREFIX} "py3plug-fd-local-fileset-basic" DISABLED
+  set_tests_properties(
+    "${SYSTEMTEST_PREFIX}py3plug-fd-local-fileset-basic:io_in_core"
+    PROPERTIES LABELS "broken"
```

Comment:
when I tried, the tests didn't fail, so are they really broken?

Reply:
Good point, I decided to remove the utf8 file in the tests for now, and will reenable them when the utf-8 handling in the python name api for python3 is fixed.
So the tests are not disabled anymore here.